### DISCUSSION TITLE
无主之地2 不能调分辨率  设置菜单不能更改内容

### DISCUSSION THREAD

**Discussion by 1322739583** [Gamuxorg/bbs#321] [游戏或网站运行反馈]
### 以下是推荐模板
游戏名称：无主之地

操作系统：*manjaro-kde 20*

bug描述：*（点修改分辨率没有反应，视频设置的其他的菜单选项也是没有反应的，而且不能按esc返回，确切的说是按任何键都没有反应 默认的分辨率是很小的，估计只有400*800* ）

终端运行输出 ：*(无)*

自行尝试解决bug的方式和结果：*（试过移除中文输入法，改成只剩下英文，并没有作用）*

**Comment by shouhuanxiaoji** [comment]
该问题已确认，暂时解决方法是手动修改配置文件。
